src/ui/Base/Initialization.ts

```diff
@@ -55,6 +56,7 @@ export class Initialization {
 
   private static logger = new Logger('Initialization');
   public static registeredComponents: String[] = [];
+  public static aliasedComponents: IStringMap<String[]> = {};
```

Comment:
How about `componentAliases` as the variable name? Using the past tense `aliased` suggests to me that an action was performed on the component.